test/visual-diff/visual-tests

```diff
@@ -690,7 +698,7 @@
     },
     {
       "flaky": true,
-      // TODO(#22162, cvializ): see https://percy.io/ampproject/amphtml/builds/1828636
+      // TODO(#22162, @ampproject/wg-ui-and-a11y): see https://percy.io/ampproject/amphtml/builds/1828636
```

Comment:
Same here. Missing `"flaky": true`?

Reply:
See line above comment :) 

(Fixing in #27461.)

Reply:
It was above the comment, moved it below for consistency

Reply:
I'm going to blame Github for hiding the previous line. Also, today is Friday 😃